pkg/bindings/generate/types.go

```diff
@@ -38,4 +38,6 @@ type SystemdOptions struct {
 	After *[]string
 	// Requires - systemd requires list for the container or pods
 	Requires *[]string
+	// AdditionalEnvVariables - environment variables setted by -e/--env
```

Comment:
This should be used a generic go library, please do not mention cli flags since they might not apply